src/i18n/config.ts

```diff
@@ -17,7 +39,7 @@ i18n
         translations: require('../locales/fr.json'),
       },
     },
-    lng: 'ar',
+    lng: window.localStorage[localStorageName] || 'ar',
```

Comment:
here

Reply:
arg, i will work on it

Reply:
barakallahu fik

Reply:
This branch was successfully deployed
1 active deployment